ddtrace/contrib/algoliasearch/__init__.py

```diff
@@ -26,7 +26,9 @@
 from ...internal.utils.importlib import require_modules
 
 
-with require_modules(["algoliasearch", "algoliasearch.version"]) as missing_modules:
+required_modules = ["algoliasearch", "algoliasearch.version"]
```

Comment:
why?

Reply:
https://github.com/DataDog/dd-trace-py/pull/3568/files#diff-6a70144455f9fea41df44b3e578cfaa81ae9c5297e41cce9ef02a5b719013817R268

so we can report what modules aren't available when we can't load the integration. This is the only integration that doesn't do the pattern.